packages/node_modules/@webex/internal-plugin-calendar/test/unit/spec/calendar.js

```diff
@@ -0,0 +1,84 @@
+/*!
+ * Copyright (c) 2015-2019 Cisco Systems, Inc. See LICENSE file.
+ */
+
+import {assert} from '@webex/test-helper-chai';
+import Calendar from '@webex/internal-plugin-calendar';
+import MockWebex from '@webex/test-helper-mock-webex';
+import btoa from 'btoa';
+import sinon from 'sinon';
+
+describe('internal-plugin-calendar', () => {
+  describe('Calendar Apis', () => {
+    const webex = new MockWebex({
```

Comment:
Could you move this initialization into the `beforeEach` block so we're sure we get a fresh webex instance for each test?

```suggestion
    let webex;
```